src/test/scala/uk/gov/nationalarchives/XMLCreatorTest.scala

```diff
@@ -126,21 +151,46 @@ class XMLCreatorTest extends AnyFlatSpec {
+    }
+
+  "createFolderOpex" should "create the correct opex xml, excluding the SourceId and Identifier if folder type is not 'ArchiveFolder', " +
+    "folder title is not blank but the name and title are the same" in {
+      val xml = XMLCreator().createFolderOpex(folder.copy(`type` = ContentFolder, title = "name"), childAssets, childFolders).unsafeRunSync()
+      xml should equal(prettyPrinter.format(expectedXMLNoHierarchyFolder))
+    }
+
+  "createFolderOpex" should "create the correct opex xml, excluding the SourceId and Identifier if folder type is not 'ArchiveFolder', " +
+    "folder title is blank and the name and title are not the same" in {
+      val xml = XMLCreator().createFolderOpex(folder.copy(`type` = ContentFolder, title = ""), childAssets, childFolders).unsafeRunSync()
+      xml should equal(prettyPrinter.format(expectedXMLNoHierarchyFolder))
+    }
 
-  "createOpex" should "use the name if the title is blank" in {
+  "createFolderOpex" should "create the correct opex xml, including the SourceId and Identifier, if folder type is 'ArchiveFolder', " +
+    "folder title is not blank and the name and title are not the same " in {
+      val xml = XMLCreator().createFolderOpex(folder, childAssets, childFolders).unsafeRunSync()
+      xml should equal(prettyPrinter.format(expectedStandardArchivedFolderXml))
+    }
+
+  "createFolderOpex" should "create the correct opex xml, using the name if the title is blank" in {
     val xml = XMLCreator().createFolderOpex(folder.copy(title = ""), childAssets, childFolders).unsafeRunSync()
-    val prettyPrinter = new PrettyPrinter(200, 2)
-    prettyPrinter.format(expectedXmlNoTitle) should equal(xml)
+    xml should equal(prettyPrinter.format(expectedXmlNoTitle))
   }
 
-  "createOpex" should "exclude the SourceId and Identifier if the name title are the same" in {
-    val xml = XMLCreator().createFolderOpex(folder.copy(title = "name"), childAssets, childFolders).unsafeRunSync()
-    val prettyPrinter = new PrettyPrinter(200, 2)
-    prettyPrinter.format(expectedXMLNoHierarchyFolder) should equal(xml)
-  }
+  "createFolderOpex" should "create the correct opex xml, excluding the SourceId and Identifier if folder type is 'ArchiveFolder', " +
+    "folder title is not blank but the name and title are the same" in {
+      val xml = XMLCreator().createFolderOpex(folder.copy(title = "name"), childAssets, childFolders).unsafeRunSync()
+      xml should equal(prettyPrinter.format(expectedXMLNoHierarchyFolder))
+    }
+
+  "createFolderOpex" should "create the correct opex xml, excluding the SourceId and Identifier if folder type is 'ArchiveFolder', " +
+    "folder title is blank and the name and title are not the same" in {
+      val xml = XMLCreator().createFolderOpex(folder.copy(title = ""), childAssets, childFolders).unsafeRunSync()
+      xml should equal(prettyPrinter.format(expectedXMLNoHierarchyFolder))
+    }
```

Comment:
Similar thing for these tests. You can get rid of any ones that are checking if the name != title and just keep the ones that check for ArchiveFolder.